### ISSUE TITLE
Unnamed Op not showing in list_tensors command during debug session

### ISSUE THREAD

**Issue by burglarhobbit** [tensorflow/tensorflow#16776]
I have these two unnamed op tensors `logits` and `outputs` under a variable scope, but the `lt` command isn't listing these two tensors under the op 'MatMul' and 'Softmax' during the `tfdbg` session after a test run on a checkpoint. Here is a snapshot of the code:
```
    with tf.variable_scope(scope):
        d_inputs = dropout(inputs, keep_prob=keep_prob, is_train=is_train)
		d_memory = dropout(memory, keep_prob=keep_prob, is_train=is_train)
		JX = tf.shape(inputs)[1]

		with tf.variable_scope("attention"):
			inputs_ = tf.nn.relu(dense(d_inputs, hidden, use_bias=False, scope="inputs"))
			memory_ = tf.nn.relu(dense(d_memory, hidden, use_bias=False, scope="memory"))
			outputs = tf.matmul(inputs_, tf.transpose(memory_, [0, 2, 1])) / (hidden ** 0.5)
			mask = tf.tile(tf.expand_dims(mask, axis=1), [1, JX, 1])
            # The tensor down below 
			logits = tf.nn.softmax(softmax_mask(outputs, mask))
            # And the tensor down below here as well
			outputs = tf.matmul(logits, memory)
			res = tf.concat([inputs, outputs], axis=2)
```
What can I do to retrieve these variables for testing purposes on `tfdbg`?  
For alternative purposes, can I retrieve them using the normal tensorflow session by using `tf.add_to_collection(op_name, tensor)` as mentioned in [this answer][1]?

  [1]: https://stackoverflow.com/questions/44639260/retrieving-an-unnamed-variable-in-tensorflow

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 109 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 124 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 139 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 154 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 169 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 184 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 199 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 214 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @caisq: It has been 229 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by harshini-gadige** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
